presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -1034,7 +1039,8 @@ private void assertFileFormat(HiveStorageFormat storageFormat)
             // no filter
             assertQueryUsingH2Cte("SELECT * FROM test_file_format_orc", cte);
             assertQueryUsingH2Cte("SELECT comment FROM test_file_format_orc", cte);
-            assertQueryUsingH2Cte("SELECT COUNT(*) FROM test_file_format_orc", cte);
+            assertQueryFails("SELECT COUNT(*) FROM test_file_format_orc", "Partial aggregation pushdown only supported for ORC/Parquet files. Set session property hive.pushdown_partial_aggregations_into_scan=false and execute query again");
```

Comment:
test_file_format_orc is Orc table, right? why partial aggregation pushdown now supported?

Reply:
The table is defined as orc - but the files are in RC or text. Surprisingly Presto supports this use case - but we cannot perform partial agg pushdown on all formats so we throw an error here

Reply:
ah... maybe add a comment, describe the case?